test/functional/test_framework/messages.py

```diff
@@ -836,22 +836,24 @@ def __repr__(self):
             % (self.challenge, self.solution)
 
 class DynaFedParamEntry:
-    __slots__ = ("m_serialize_type", "m_signblockscript", "m_signblock_witness_limit", "m_fedpeg_program", "m_fedpegscript", "m_extension_space")
+    __slots__ = ("m_serialize_type", "m_signblockscript", "m_signblock_witness_limit", "m_fedpeg_program", "m_fedpegscript", "m_extension_space", "m_extra_root")
```

Comment:
So `elided_root` is called `extra_root` in Python?

Reply:
what name do you like more

Reply:
I would just have called it elided_root like in the C++ impl.